build/commands/scripts/sync.js

```diff
@@ -12,7 +12,6 @@ const os = require('os')
 
 program
   .version(process.env.npm_package_version)
-  .arguments('[ref]')
```

Comment:
We need to also update the docs:
https://github.com/brave/brave-browser#update-brave

Reply:
This is still outstanding @petemill.